### DISCUSSION TITLE
Self host configuration options and incoming "plus tagged" email questions

### DISCUSSION THREAD

**Discussion by FreelancerJay** [simple-login/app#2329] [Self-hosting question] [answered]
Reading the docs, I keep seeming to cross between info about the hosted service in conjunction with protonmail, and the self-hosted capabilities. So I'm hoping these are easy questions to answer. Just need a "yes it can" or "no it can't". Don't need the "here is how", I'm happy to keep digging in the docs and source (though you are so inclined to how to, I won't say no, I'd appreciate pointers to relevant docs so I can learn and become a helpful community member too)

In a self hosting situation, can Simple-Login:
a) be part of your primary email server, basically taking over alias management? (so users can then use a single email address in their mail client and not have to add every alias to every client)
b) handle both a primary domain and a secondary domain (so aliases for both @myname.com, the real domain, and @fakename.com, the one to give to businesses you don't trust with your real email server or name)

If a) is yes, then can Simple-Login:
1) deliver incoming emails directly to the inbox of the target address, just updating the "reply-to:" field for the alias and sender details, rather than sending it out via smtp to itself and modifying the "from:"?
2) put a copy of outbound emails in the user's Sent mailbox _after_ substituting the "To:" and "From:" fields for the alias and reverse-alias info?

Lastly, basically unrelated to the above, how does Simple-Login handle when the sender address uses a + tag? there are a few companies I know of that when you are emailing their support, make their "from:" address follow a pattern along the lines of "support+<casenumber/reference>@domain.com" so I'm a little curious how the reverse-alias system works with that.

Thanks for all your time!

**Comment by nguyenkims** [comment, marked as answer]
I think the answer is NO for both an and b, at least with a simple setup.

**Reply by FreelancerJay** [reply]
Thanks for the quick reply.
If I don't find anything else that works for this, I'll be back :-)
